common/oatbox/filesystem/FileSystemHandler.php

```diff
@@ -22,13 +22,18 @@
 
 namespace oat\oatbox\filesystem;
 
+use oat\oatbox\service\ServiceManager;
+use ReflectionClass;
+use ReflectionProperty;
 use Zend\ServiceManager\ServiceLocatorAwareInterface;
 use Zend\ServiceManager\ServiceLocatorAwareTrait;
 
 abstract class FileSystemHandler implements ServiceLocatorAwareInterface
 {
     use ServiceLocatorAwareTrait;
 
+    protected const UNSERIALIZABLE_PROPERTIES = ['fileSystem', 'serviceLocator'];
```

Comment:
I think it's better to keep this one private.
If a descendant class would want to add its own unserializable properties, it's IMO better if it would override the `__sleep()` method. That way it will not have to rely on _internal_ implementation details of this abstract class.